scripts/background-tasks.sh

```diff
@@ -53,14 +48,15 @@ fi
 
 exitcode=$?
 if [ $exitcode -ne 0 ]; then
-    echo "Cohesion import failed. Website will remain in maintenance mode."
-    # In certain scenarios ACSF can keep a site live even if db-update.sh results in an error.
-    set_maintenance_mode 1
-    set_site_owner_maintenance_mode 1
+  echo "Cohesion import failed. Website will remain in maintenance mode."
```

Comment:
For a script being called background-tasks.sh I would not expect to see Cohesion mentioned in the file